src/applications/disability-benefits/2346/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,109 @@
+import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import { focusElement } from 'platform/utilities/ui';
+import React from 'react';
+
+class IntroductionPage extends React.Component {
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/11679/files#r377906116

Reply:
We will stay with `React.Component` for now because that is default when we generate the form.